anyblok/tests/test_many2many.py

```diff
@@ -64,6 +65,34 @@ class Person:
                               many2many="persons")
 
 
+def _complete_many2many_with_ordered_backref(**kwargs):
+
+    @register(Model)
+    class Address:
+
+        id = Integer(primary_key=True)
+        street = String()
+        zip = String()
+        city = String()
+
+    @register(Model)
+    class Person:
+
+        name = String(primary_key=True)
+        addresses = Many2Many(model=Model.Address,
+                              join_table="join_addresses_by_persons",
+                              remote_columns="id", local_columns="name",
+                              m2m_remote_columns='a_id',
+                              m2m_local_columns='p_name',
+                              many2many=(
+                                  "persons",
+                                  dict(
+                                      order_by="ModelPerson.name",
+                                      collection_class=ordering_list('name'),
+                                  ),
+                              ))
+
```

Comment:
Create a Person and check if addresses is an ordering list and it will failed.
You can try with something like this : 
```
bob = Person.insert(name="Bob")
assert bob.addresses.ordering_attr
```